recipe/meta.yaml

```
@@ -10,25 +10,27 @@ source:
   git_url:  https://github.com/pyg-team/pyg-lib/
   git_rev: {{ version }}
 
+  patches:
+    - fix-cmake.patch
+
 build:
-  number: 1
+  number: 10
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
   # Only build for CUDA >=11.2
   skip: true  # [win or cuda_compiler_version in ("10.2", "11.0", "11.1")]
-
+  
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}                 # [cuda_compiler_version not in (undefined, 'None')]
-    - sysroot_linux-64 ==2.17                # [linux64]
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pytorch                                # [build_platform != target_platform]
-    - pytorch =*={{ torch_proc_type }}*      # [build_platform != target_platform]
+    - {{ compiler('cuda') }}                             # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_linux-64 ==2.17                            # [linux64]
+    - python                                             # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}                 # [build_platform != target_platform]
+    - pytorch ={{ pytorch }}={{ torch_proc_type }}*      # [build_platform != target_platform]
     - cmake
-    - ninja
+    - ninja  # NOTE(hadim): here ninja is used for linux and osx
```

What's the purpose of this note? If it's only used on unix, then do:
```suggestion
    - ninja        # [unix]
    - llvm-openmp  # [osx]
```